ui/tests/e2e/test.js

```diff
@@ -16,6 +17,9 @@ describe("End-to-end", () => {
     // Click first Age facet value
     let facetValueRow = await getFacetValueRow("Age", "10-19");
     await facetValueRow.click("input");
+    // After data is returned from backend, Smoker facet will have "107".
```

Comment:
For `expect(grayDiv).toBeTruthy();`, is this assertion necessary? Won't the test fail anyway if that div doesn't become gray?

Reply:
How would the test fail? If there's a bug where div doesn't become gray, grayDiv will be null and test will pass.